datafusion/core/src/datasource/physical_plan/parquet/statistics.rs

```diff
@@ -109,17 +109,24 @@ macro_rules! get_statistic {
                     }
                 }
             }
-            // type not supported yet
+            // type not fully supported yet
             ParquetStatistics::FixedLenByteArray(s) => {
                 match $target_arrow_type {
-                    // just support the decimal data type
+                    // just support specific logical data types, there are others each
+                    // with their own ordering
                     Some(DataType::Decimal128(precision, scale)) => {
                         Some(ScalarValue::Decimal128(
                             Some(from_bytes_to_i128(s.$bytes_func())),
                             *precision,
                             *scale,
                         ))
                     }
+                    Some(DataType::FixedSizeBinary(size)) => {
+                        Some(ScalarValue::FixedSizeBinary(
```

Comment:
Should we verify that the resulting bytes have the correct size? As in verify that `*size` is the same as `(s.$bytes_func().to_vec().len()`?

If not, I suggest we ignore the error (but don't use the statistics) the same way as is done for non UTF8 values i string statistics. Perhaps we can at least debug when this happens with `debug!`

Since this data can come from any arbitrary parquet writer, I think it is good practice to

Reply:
Oh sure, good catch.

The code for strings is:

```
let s = std::str::from_utf8(s.$bytes_func())
    .map(|s| s.to_string())
    .ok();
Some(ScalarValue::Utf8(s))
```

which looks like it's treated as a NULL, rather than ignored as a statistics value. Should I do the same here?

Reply:
Yes, I think converting to `NULL` is the correct behavior. In this case that results in ignoring the value in terms of range analysis. More details are here if you are curious:

https://github.com/apache/arrow-datafusion/blob/8074ca1e758470319699a562074290906003b312/datafusion/core/src/physical_optimizer/pruning.rs#L85-L87

Reply:
I see, done. I also added a log statement for strings, for consistency.